MO4/Sniffs/Arrays/ArrayDoubleArrowAlignmentSniff.php

```diff
@@ -122,11 +138,17 @@ public function process(File $phpcsFile, $stackPtr): void
                 }
 
                 if ($hasKeyInLine === false) {
-                    $phpcsFile->addError(
+                    $fixable = $phpcsFile->addFixableError(
                         'in arrays, keys and "=>" must be on the same line',
                         $i,
                         'KeyAndValueNotOnSameLine'
                     );
+
+                    if ($fixable === true) {
+                        $phpcsFile->fixer->beginChangeset();
+                        $phpcsFile->fixer->replaceToken($j, '');
```

Comment:
```diff
@@ -146,7 +146,7 @@ class ArrayDoubleArrowAlignmentSniff implements Sniff
 
                     if ($fixable === true) {
                         $phpcsFile->fixer->beginChangeset();
-                        $phpcsFile->fixer->replaceToken($j, '');
+                        $phpcsFile->fixer->replaceToken((int) $j, '');
                         $phpcsFile->fixer->endChangeset();
                     }
                 }
```

Reply:
I'll cast `$j` to `int` when it's get declared.